8Knot/pages/visualization_template/viz_template.py

```diff
@@ -1,13 +1,10 @@
-from dash import html, dcc, callback
-import dash
+from dash import dcc, callback
 import dash_bootstrap_components as dbc
-from dash.dependencies import Input, Output, State
-import plotly.graph_objects as go
```

Comment:
I think this one should stay as it could still show up on this file vs the VisualizationAIO